package.json

```
@@ -7,7 +7,7 @@
     "type": "git",
     "url": "https://github.com/onfido/onfido-sdk-ui.git"
   },
-  "license": "MIT",
+  "license": "SEE LICENSE IN onfido-sdk-ui/LICENSE",
```

Minor: I think this should be
```suggestion
  "license": "SEE LICENSE IN ./LICENSE",
```